core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2097,86 +2097,87 @@ package object config {
 
   private[spark] val PUSH_BASED_SHUFFLE_ENABLED =
     ConfigBuilder("spark.shuffle.push.enabled")
-      .doc("Set to 'true' to enable push-based shuffle on the client side and this works in " +
+      .doc("Set to true to enable push-based shuffle on the client side and this works in " +
         "conjunction with the server side flag spark.shuffle.server.mergedShuffleFileManagerImpl " +
         "which needs to be set with the appropriate " +
         "org.apache.spark.network.shuffle.MergedShuffleFileManager implementation for push-based " +
         "shuffle to be enabled")
-      .version("3.1.0")
+      .version("3.2.0")
```

Comment:
It seems that we have exposed this in 3.1.0(3.1.1) already: https://github.com/apache/spark/blob/branch-3.1/core/src/main/scala/org/apache/spark/internal/config/package.scala

Why do we change the version here? The configuration is only effective since 3.2?

Reply:
Yes, the feature was not ready in 3.1. It will be ready only in 3.2 which is why I changed it to 3.2 to keep it consistent.